app/domain/errors.rb

```diff
@@ -292,16 +292,21 @@ module AuthnAzure
           code: "CONJ00050E"
         )
 
-        ClaimInInvalidFormat = ::Util::TrackableErrorClass.new(
-          msg:  "xms_mirid claim has been received in an invalid format. Reason: {0}",
+        TokenFieldNotFoundOrEmpty = ::Util::TrackableErrorClass.new(
+          msg:  "Field '{0-field-name}' not found or empty in token",
```

Comment:
not found or empty token

Reply:
what do you mean?

Reply:
need to remove the word "in" (empty **in** token)

Reply:
but it should be there :)

the field is missing inside the token. the token itself is not missing.